src/ParkChecklist/ParkChecklist.js

```diff
@@ -7,222 +7,226 @@ import page from "../resources/page.png";
 import SelectedStateParks from "../SelectedStateParks/SelectedStateParks";
 import ParkDetails from "../ParkDetails/ParkDetails";
 
-const ParkChecklist = ({ apiLink, indivParkLink }) => {
-    const [loading, setLoading] = useState(true);
-    const [parks, setParks] = useState([]);
-    const [park_code, setParkCode] = useState(null);
-    const [checkedItems, setCheckedItems] = useState([]);
-    const [selectedState, setSelectedState] = useState("");
-    const [showResults, setShowResults] = useState(false);
-    const [foundPark, setFoundPark] = useState(null);
-    const [currentPage, setCurrentPage] = useState(1);
-    const navigate = useNavigate();
-    const parksPerPage = 48;
-    const states = [
-        "AL",
-        "AK",
-        "AZ",
-        "AR",
-        "CA",
-        "CO",
-        "CT",
-        "DE",
-        "FL",
-        "GA",
-        "HI",
-        "ID",
-        "IL",
-        "IN",
-        "IA",
-        "KS",
-        "KY",
-        "LA",
-        "ME",
-        "MD",
-        "MA",
-        "MI",
-        "MN",
-        "MS",
-        "MO",
-        "MT",
-        "NE",
-        "NV",
-        "NH",
-        "NJ",
-        "NM",
-        "NY",
-        "NC",
-        "ND",
-        "OH",
-        "OK",
-        "OR",
-        "PA",
-        "RI",
-        "SC",
-        "SD",
-        "TN",
-        "TX",
-        "UT",
-        "VT",
-        "VA",
-        "WA",
-        "WV",
-        "WI",
-        "WY",
-    ];
+const ParkChecklist = ({ apiLink }) => {
+  const [loading, setLoading] = useState(true);
+  const [parks, setParks] = useState([]);
+  const [park_code, setParkCode] = useState(null);
+  const [checkedItems, setCheckedItems] = useState([]);
+  const [selectedState, setSelectedState] = useState("");
+  const [showResults, setShowResults] = useState(false);
+  const [foundPark, setFoundPark] = useState(null);
+  const [currentPage, setCurrentPage] = useState(1);
+  const navigate = useNavigate();
+  const parksPerPage = 48;
+  const states = [
+    "AL",
+    "AK",
+    "AZ",
+    "AR",
+    "CA",
+    "CO",
+    "CT",
+    "DE",
+    "FL",
+    "GA",
+    "HI",
+    "ID",
+    "IL",
+    "IN",
+    "IA",
+    "KS",
+    "KY",
+    "LA",
+    "ME",
+    "MD",
+    "MA",
+    "MI",
+    "MN",
+    "MS",
+    "MO",
+    "MT",
+    "NE",
+    "NV",
+    "NH",
+    "NJ",
+    "NM",
+    "NY",
+    "NC",
+    "ND",
+    "OH",
+    "OK",
+    "OR",
+    "PA",
+    "RI",
+    "SC",
+    "SD",
+    "TN",
+    "TX",
+    "UT",
+    "VT",
+    "VA",
+    "WA",
+    "WV",
+    "WI",
+    "WY",
+  ];
 
-    const fetchParkDetails = async (park_code) => {
-        try {
-            const response = await axios.get(`${apiLink}/${park_code}`);
-            const selectedPark = response.data.data;
+  const fetchData = async () => {
+    try {
+      const response = await axios.get(apiLink);
+      setParks(response.data.data);
```

Comment:
I like the use of getting rid of our error console in favor of displaying the message to the user, it makes the design better